src/components/SearchResults/RequestSearchForm.js

```diff
@@ -25,17 +40,29 @@ export const RequestSearchForm = ({ closeForm, queryTerm = '' }) => {
           pediatric_cancer: '',
           approach: '',
           email: '',
-          email_updates: false
+          email_updates: false,
+          query: queryTerm,
```

Comment:
Is there a reason why we pass this as a prop and grab it inside the component?

Reply:
Initially I passed the search term value from the parent component, but it has been removed. Now the search term value is retrieved using `useRouter` from `RequestSearchFrom` , pls see the commit at 713810f1f41b64fec23b9a6218012958d8f742bd.